community-membership.md

```diff
@@ -86,6 +86,14 @@ List of contributions:
 
 ```
 
+### Kubernetes Ecosystem
+
+There are related Kubernetes GitHub organizations, such as [Kubernetes-sigs]. We are currently working on
```

Comment:
nit: the org name itself is lower case `kubernetes-sigs`. This would need to be fixed with the link on L247 too